### DISCUSSION TITLE
Is there OvenMediaEngine server API functionality for streaming?

### DISCUSSION THREAD

**Discussion by ihorafanasiev** [OvenMediaLabs/OvenMediaEngine#1544] [Help] [answered]
Can OvenMediaEngine be used as a server for broadcasting an RTP stream from ffmpeg to a Webrtc browser client?  
Like FFmpeg ---RTP---> OvenMediaEngine ---Webrtc--->only one browser client
And there can be several such flow schemes, and the number of this flow is dynamically adjusted.
Does the server support this feature?
I did not find a similar configuration in the API documentation.
Thanks

**Comment by getroot** [comment, marked as answer]
Is your question correct about whether it is possible to ingest pure RTP without signaling?

Protocols transmitted stream using RTP, such as RTSP and WebRTC, are supported, but transmission only over RTP is not supported and is almost impossible. What makes it possible is RTSP or WebRTC.

And if the sender uses ffmpeg you can choose SRT, RTMP or MPEG-2 TS/udp instead of RTP. And that's a much better option.
